addons/portal/static/src/scss/portal.scss

```diff
@@ -206,7 +206,6 @@ li > p {
     border-bottom-width: $border-width;
     border-bottom-style: solid;
     border-bottom-color: $border-color;
-    line-height: 2.1rem;
```

Comment:
Yes, can be removed 👍 

There is a bit of Odoo history  behind this rule. I guess that it has been introduced to handle multi-lines design for elements that were initially using the glorious `page-header` class, discontinued since BTS v3 (https://getbootstrap.com/2.3.2/components.html#typography) and then replaced with our custom `o_page_header`.

It was coded in a time where themes CSS was much more "aggressive" against the default design... at the time it was probably overridden by themes.

Reply:
@stefanorigano  Thanks for the explanation and check!